packages/cli/src/commands/dev.ts

```diff
@@ -21,7 +21,7 @@ export const DevCommandOptionsSchema = z.object({
   port: z.coerce.number(),
   envFile: z.string(),
   handlerPath: z.string(),
-  clientId: z.string().optional(),
+  clientId: z.string(),
```

Comment:
This change will break the dev command. 

You can do `type WhoAmIOptions = Required<DevCommandOptions>` or just create a new type for WhoAmI.

Reply:
@matt-aitken  Yeah, I realized that later, check the commit below that one named "revert zod schema for dev.ts"

Reply:
The latest code has still made this optional

Reply:
I fixed it